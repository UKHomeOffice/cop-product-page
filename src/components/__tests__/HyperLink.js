import React from "react"
import renderer from "react-test-renderer"
import HyperLink from "../hyperlink";

describe("HyperLink", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<HyperLink/>)
      .toJSON();
    expect(tree).toMatchSnapshot()
  })
})