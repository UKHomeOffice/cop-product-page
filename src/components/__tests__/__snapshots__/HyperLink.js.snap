// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HyperLink renders correctly 1`] = `
<a
  className="govuk-link"
/>
`;
